### ISSUE TITLE
Google photo url generation

### ISSUE THREAD

**Issue by Martouf0208** [pibooth/pibooth-qrcode#23]
I'm trying to use the url generate with google-photo plugins, the problem is the url is super long and so the generated qrcode also.

In my config file I use :
_prefix_url = {url}_

The value of url in my case is :
https://lh3.googleusercontent.com/lr/AAJ1LKcPJijEpU2II7ZDw8ypP3rQQlyEqxk-eNM49Fj1GqldC1tRhjH4aBEAS1uKh-FVHMrBB2JqfvdfwSsbJBXtpbExmR0TWEZjHK0jaGnNmeWlX-mgQd_2i3zB0OFDPBekn8Ic6CKiuVcZKtpLt-qEll18XGz6BAYD22T9n3D68N29GA2EvqdnOzFlpWNyFgz49-OAodaYG8MmGn6afD7vORKP_F5YgGoYgwBlbWcLPaa4SamEJntrBnvkxgFo1Uj_aUkvvjadTuZRxMbHJEADQnSFqYABx89xQ1jicA1i_5Pm2ssJcBxgcuWv1YvK8dHNi9BGkCdqWPJGcw8d4ubQrihwD4iMHkXxU-L5jAtMq4M9kGnb3pAfiD6ZZAfuoHfWSPE0KEL9ALUjRyJOC3-CK8ThMsCG2d16pZ0S06QB4q-y8wY_xRCKUiZ4D6OWzPAKMEfKQywdJ6uldVxdmAspjRbTAHGlmS7E9gOsjPOOzYUFuAgKGyfbuXXKSHRXhKxSeZ9wvNwDsbFtvhVN56E5VcgsdvOY1lsYnbZi5zjhzRnWg9hXqy1fRTrAPs68EWR4RATtD32w7mQTLr8EEI2FfxwRi91OMQTdJhne0jStQQDmgJ7-ra1oGGH0E4QSynuHuNKqTI15xG8yI7dVYFCIwgeFX4y0jrqWrC01-7IUKryJ8r2NA9CpN2c6B6CYu6m1KpVVYTgZlhlbAo_q7D9c8lvpMn6blObZqltctxyhuz_4CvIy3btmwrX4e-11m0PXzgBAolNccFQA7Q5MHbSPHZl0pi_mSKebhV8wkYaQp04FPlk5-UBWurjvqbCNp2QjLUZG5PfdAQwUkaJNclhqbUY4GjJ6jUmSeRkjWu4D70zZzZNPLLH9VnZx0QFNfOF9rTSbuuxV94fdLCEzGcZh1yF0N9w6YQIYxtBPHC7ahW8VtDuZq1bKq_NbxtnDqFREDIcpZBc7OePHjFf4kgpPnNZ4ABEdLqBuSho4HSYX3PjIdDH0T_WcLUJEWo7ThHCtReIkebUyxPiT5Rm-GRlkHrUT8AZc7NQ

If someone could help me 🙏

**Comment by Hexidecibel** [comment]
Hi there, not my project but I was just able to get this working by simply using {url} for the url.  I believe the pibooth-google-photo plugin sets this value and you can use {url} so inject the last uploaded photos url as the qrcode

**Comment by Martouf0208** [comment]
Yes it works well but the problem is the size of the qrcode it seem to depends on the longer of the url so the qrcode is really big and takes time to be generated

**Comment by frankchau93** [comment]
Yeah, the QR code gets way too big. I wonder if there's a better way to shorten it

**Comment by sebaule** [comment]
hello ! I have made a plugin in order to reduce this url !
you have to force the google-photo plugin to start the first and execute my plugin and after this qrcode plugin is executed (by default that)
I will send you my plugin ! I have to push it on my git

**Comment by sebaule** [comment]
Here the code of my plugin and the methode to add it inside pibooth :) 
[pibooth_photo_share](https://github.com/sebaule/pibooth_photo_share)
Regards !
